search-ui/src/views/BusinessInfoView.vue

```diff
@@ -258,10 +258,25 @@ const loadPurchasableDocs = async () => {
         'can only be ordered if the business is in Good Standing.' : ''
     })
   }
+
+  if (isCstatAvailable()) {
+    purchasableDocs.value.push({
+        code: FeeCodes.CSTAT,
+        fee: displayFee(feeData[2].fee, false),
+        label: 'Certificate of Status',
+        documentType: DocumentType.CERTIFICATE_OF_STATUS,
+        active: true,
+        tooltip: ''
+    })
+  }
 }
 
 const isCogsAvailable = () => {
-  return !isFirm.value
+  return !isFirm.value && isActive.value
+}
+
+const isCstatAvailable = () => {
+  return (isBC.value || isCoop.value) && isActive.value
```

Comment:
should this include BENs too?

Reply:
BEN is not in the list provided

Reply:
Oh weird. I thought BENs were essentially the exact same as BCs except for the benefit statement

Reply:
I will confirm it with Jenny